Makefile

```diff
@@ -330,6 +331,8 @@ e2e-install:
 		--create-namespace \
 		--set 'manager.image.pullPolicy=Never' \
 		--set 'manager.resources=null'\
+		--set "manager.image.registry=$(REGISTRY)" \
+		--set "manager.image.repository=$(REPOSITORY)" \
```

Comment:
Is there a reason you added this?

Reply:
I was testing my fork with my own registry, and the `e2e` flow with helm install does not support the REGISTRY and REPOSITORY env variables when set.
If i do 
```
export REGISTRY=docker.io
export REPOSITORY=gchiesa/capsuleproject/capsule
make e2e/1.27.2
```
I was getting an error because the image was build and published but  helm release was still trying to deploy the capsule upstream image coordinates.